libs/ledger-live-common/src/data/icons/svg/PTRN.svg

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg id="Layer_2" data-name="Layer 2" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 19.33 23.28">
+  <defs>
+    <style>
+      .cls-1 {
+        fill: #273658;
+      }
+    </style>
+  </defs>
+  <g id="Layer_1-2" data-name="Layer 1">
```

Comment:
we must not have id or class in the SVG. these will not work for React Native integration.
see other icons for example.
There might be an automatic tool out there to sanitize the SVG but we must not have defs, style, classes or global ids.

Thanks

Reply:
Thanks for the info. I will fix it and make a new commit.